src/app/api/workout/[workoutId]/favorite/route.ts

```
@@ -0,0 +1,55 @@
+      where: {
+        id: workoutId,
+        userId
+      }
+    })
+
+    if (!existingWorkout) {
+      return new NextResponse('Workout not found', { status: 404 })
+    }
+    
+    // TypeScript workaround - cast to any to handle new field
+    const currentFavoriteStatus = (existingWorkout as any).favorite || false
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)